app/controllers/concerns/form_attachment_create.rb

```diff
@@ -11,10 +11,9 @@ def create
     form_attachment_model = self.class::FORM_ATTACHMENT_MODEL
     form_attachment = form_attachment_model.new
     namespace = form_attachment_model.to_s.underscore.split('/').last
+    filtered_params = params.require(namespace).permit(:file_data)
 
-    file_data = params.require(namespace).permit(:file_data)
-
-    form_attachment.set_file_data!(file_data)
+    form_attachment.set_file_data!(filtered_params[:file_data])
```

Comment:
@omgitsbillryan Should have been `filtered_params[namespace][:file_data]`, right?

Reply:
ah crap. ya that's a pretty important detail that we overlooked

Reply:
actually it works...
<img width="1320" alt="Screen Shot 2020-01-08 at 7 55 07 PM" src="https://user-images.githubusercontent.com/19188/72028514-d0330b00-3250-11ea-9971-501e68d51d2b.png">